apps/onboarding-skill/src/messaging/MessageDispatcher.ts

```diff
@@ -70,9 +70,12 @@ class MessageDispatcher implements IMessageDispatcher {
     submodel: Submodel
   ) {
     //TODO: an error here leads to unhandled rejection
-    this.messageSender.replyTo(message.frame, MessageTypes.RESPONSE_INSTANCE, [
-      submodel
-    ]);
+    this.sendNextMessageInConversationTo(
+      "Operator",
```

Comment:
How is the receiver specified?

Reply:
In a more generic case  the sender of the original approval request should be used from the original state maybe?

Reply:
This was changed from the more generic case (Initiator) to the specific case because implementing the generic case is more complicated and error-prone as the the original sender needs to be kept track of. This was not being done and was a bug. Our current state machine does not support this and implementing it would require implementation effort on that side as well as the persistence side with all the unit tests required. Since it is currently not a requirement to have initiators other than the operator it is better to keep it simple and make it explicit.

Reply:
Ok, I see. I guess the skill is also specific to the onboarding use-case. 
Perhaps if later there is a standardised task for approval processes, this could be implemented in its own skill